docs/especificacao.md

```diff
@@ -262,42 +262,67 @@ c)  Descreva o fluxo alternativo caso houver... <br>
-b)  O sistema exibe uma mensagem de erro informando quais campos precisam ser corrigidos. <br>
-c)  O patrocinador ajusta as informações e tenta salvar novamente. <br>
+a)  O patrocinador insere dados inválidos, como um formato de e-mail incorreto ou um número de telefone incompleto. <br/>
+b)  O sistema exibe uma mensagem de erro informando quais campos precisam ser corrigidos. <br/>
+c)  O patrocinador ajusta as informações e tenta salvar novamente. <br/>
 
 **Pós-condições**: O perfil do patrocinador é atualizado com sucesso no sistema.
 
 #### Gerenciar as demandas (CSU07) DAVIH
 
-**Sumário**: Um breve resumo do que o caso de uso faz e qual o seu objetivo principal.
+**Sumário**: O Representante da ONG realiza a gestão (inclusão, remoção, alteração e consulta) dos dados sobre as demandas. O objetivo principal é garantir que o Representante tenha controle sobre suas demandas dentro do sistema.
 
-**Ator Primário**: Adicione o ator principal.
+**Ator Primário:** Representante da ONG.
 
-**Ator Secundário**: Adicione o ator secundário.
+**Ator Secundário:** Não possui.
 
-**Pré-condições**: Condições que devem ser atendidas antes que o caso de uso possa ser executado.
+**Pré-condições:** O Representante deve estar autenticado e validado pelo sistema.
 
 **Fluxo Principal**:
-
-1)  Coloque aqui o fluxo.
-2)  Coloque aqui o fluxo.
-3)  Coloque aqui o fluxo.
-4)  Coloque aqui o fluxo...
-
-**Fluxo Alternativo**: Coloque aqui o nome fluxo alternativo caso houver.
-
-a)  Descreva o fluxo alternativo caso houver. <br>
-b)  Descreva o fluxo alternativo caso houver. <br>
-c)  Descreva o fluxo alternativo caso houver... <br>
-
-**Fluxo Alternativo**: Coloque aqui o nome fluxo alternativo caso houver.
-
-a)  Descreva o fluxo alternativo caso houver. <br>
-b)  Descreva o fluxo alternativo caso houver. <br>
-c)  Descreva o fluxo alternativo caso houver... <br>
-
-**Pós-condições**: O que deve ser verdade depois que o caso de uso é concluído com sucesso.
+1) 	O Representante requisita manutenção de demandas.
+2) 	O Sistema apresenta as operações que podem ser realizadas: inclusão de uma nova demanda, alteração de uma demanda, a exclusão de uma demanda e a consulta de dados de uma demanda.
+3) 	O Representante seleciona a operação desejada: Inclusão, Exclusão, Alteração ou Consulta, ou opta por finalizar o caso de uso.
+4) 	Se o Representante desejar continuar com a gestão de demandas, o caso de uso retorna ao passo 2; caso contrário o caso de uso termina.
+
+### Fluxo Alternativo (3): Inclusão
+A) O Representante requisita a inclusão de uma nova demanda. <br/>
+B) O Sistema apresenta um formulário solicitando o nome, tipo e descrição da demanda. <br/>
+C) O Representante preenche o formulário com as informações solicitadas. <br/>
+D) O Sistema valida os dados fornecidos:
+  * Se os dados estiverem corretos, o Sistema inclui a nova demanda e notifica o sucesso da operação.
+  * Se houver erros, o Sistema solicita correções e repete a validação.
+    
```

Comment:
Davih, não vejo necessidade em adicionar bullet points aqui. 
Poderia também formatar o texto para deixar padronizado com o restante do documento, por favor?
Por exemplo, deixar as letras minúsculas "a) , b), c)" retirar o número do fluxo alternativo (3). Se for numerar todos vai dar muito trabalho, e não é imprescindível ser numerado.  

Esse comentário se aplica ao restante das alterações.  :)

Reply:
Acabei de dar uma editada em geral aqui! Se deixei passar algum detalhe ou algo ainda não faz sentido, me avisa que eu volto e corrijo sem problemas.

Bom dia, boa tarde e boa noite lindões.